README.md

```diff
@@ -0,0 +1,30 @@
+# estella-two-column-dnd
+
+> Made with create-react-library
+
+[![NPM](https://img.shields.io/npm/v/estella-two-column-dnd.svg)](https://www.npmjs.com/package/estella-two-column-dnd) [![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
```

Comment:
Można by zmienić readmie skoro jest to biblioteka i ktoś przez przypadek z niej próbował korzystać poza nami